src/main/java/codeu/controller/ActivityServlet.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet class responsible for the activity page. */
+public class ActivityServlet extends HttpServlet {
```

Comment:
Could you create an ActivityServletTest as well, even if all it does is verify that the code runs without crashing?  This way, as you add functionality to this it will be easy to add unit tests for it.

Reply:
I have added a simple test but it seems to failing. Do you see anything wrong?

Reply:
Never mind, I found the error and pushed the test to my repository.